<launch>
  <node pkg="rosserial_python" type="serial_node.py" name="arduino1">
    <param name="port" value="/dev/ttyACM0"/>
  </node>

  <node pkg="state_machine" type="easy_walk.py" name="easy_walk" output="screen"/>
  <node pkg="rqt_smach" type="rqt_smach" name="rqt_smach"/>
  <node pkg="rqt_graph" type="rqt_graph" name="rqt_graph"/>
</launch>
